forge-gui/res/cardsfolder/upcoming/the_second_doctor.txt

```diff
@@ -0,0 +1,12 @@
+Name:The Second Doctor
+ManaCost:2 W U
+Types:Legendary Creature Time Lord Doctor
+PT:2/4
+S:Mode$ Continuous | Affected$ Player | SetMaxHandSize$ Unlimited | Description$ Players have no maximum hand size.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChoose | TriggerDescription$ How Civil of You — At the beginning of your end step, each player may draw a card. Each opponent who does can't attack you or permanents you control during their next turn.
+SVar:TrigChoose:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBAllowDraw
```

Comment:
shouldn't need Repeat at all
just let all draw

Reply:
If I understand this card correctly every player can choose for itself if they want to draw ?

Reply:
sure